src/main/java/core/basesyntax/operation/BalanceOperation.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax.operation;
```

Comment:
The package declaration should not be followed by an empty line. Please remove the redundant empty line after the package statement.